CMakeLists.txt

```diff
@@ -31,15 +31,24 @@ find_package(Threads REQUIRED)
 target_link_libraries(fdeep INTERFACE Threads::Threads)
 
 hunter_add_package(FunctionalPlus) # default noop
-find_package(FunctionalPlus CONFIG REQUIRED)
-target_link_libraries(fdeep INTERFACE FunctionalPlus::fplus)
+if(TARGET fdeep)                   # if used via subdirectory
+  # Note: Sadly, FunctionalPlus does not provide an alias for use as a subdirectory
```

Comment:
Thanks to you [it now does](https://github.com/Dobiasd/FunctionalPlus/pull/174). :slightly_smiling_face:

Reply:
When should this be changed here? After a release, now, etc?

Reply:
Since frugally-deep generally recommends using the latest version of FunctionalPlus from the master branch, I think it's OK to already adjust this now, even without a new release of FunctionalPlus.

Nevertheless, I'll release a new version of FunctionalPlus in a few minutes (one the CI has succeeded there).

Reply:
Updated